app/mailers/mailer.rb

```diff
@@ -1,11 +1,8 @@
-class CommunicartMailer < ApplicationMailer
+class Mailer < ApplicationMailer
```

Comment:
> It is generic

It's generic, but only for notifications around Proposals. The `ApplicationMailer` is the generic base mailer class, which is what all of our mailers (including ones that aren't Proposal-centric, like the `FeedbackMailer`) have as an ancestor. The fact that `Mailer` is a subclass of `ApplicationMailer` here is strange to me.

> I agree that we should move more methods out of this and into smaller mail classes but was trying to keep this PR under 400 changes

:+1: